java/gradle-plugin/README.md

```diff
@@ -1,7 +1,7 @@
 To build with a development version of this plugin, enter the directory
 containing this README and repeat as follows:
-* make changes
-* gradle build
-* mvn install:install-file -Dfile=build/libs/bond-gradle-1.0.jar -DgroupId=org.bondlib -DartifactId=gradle -Dversion=1.0 -Dpackaging=jar
+* write some code
+* gradle build install
```

Comment:
Consider using \` tick marks \` around `gradle build install` and the `mvn` command, as they're "computer words".